dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/ProcessingHint.java

```diff
@@ -34,5 +34,6 @@
  */
 public enum ProcessingHint
 {
-    SINGLE_INDICATOR_REPORTING_RATE_FILTER_ITEM;
+    SINGLE_INDICATOR_REPORTING_RATE_FILTER_ITEM,
```

Comment:
Just reuse the existing `SINGLE_INDICATOR_REPORTING_RATE_FILTER_ITEM` value,  they represent the same situation.